src/applications/facility-locator/components/AppointmentInfo.jsx

```diff
@@ -113,9 +113,9 @@ export default class AppointmentInfo extends Component {
         );
 
       return (
-        <Fragment>
+        <>
```

Comment:
This was done in reference to: https://github.com/department-of-veterans-affairs/vets-website/pull/10780#discussion_r326361499 @rianfowler